net.xy.ykhwong.dosbox-daum.yml

```diff
@@ -0,0 +1,92 @@
+app-id: net.xy.ykhwong.dosbox-daum
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: dosbox
+build-options:
+  strip: true
```

Comment:
```suggestion
```

Reply:
I'll try that again, but I had issue with it not working correctly.

Reply:
removing that strip: true breaks the build:

stripping /home/eben/daum/flathub/.flatpak-builder/rofiles/rofiles-6nb3P2/files/lib/concurrent_hash_map.o to /home/eben/daum/flathub/.flatpak-builder/rofiles/rofiles-6nb3P2/files/lib/debug/lib/concurrent_hash_map.o.debug
Error: module libtbb: Child process killed by signal 11

Reply:
> 

I really can't make this change, well at least not in a way that I know how to stop the build from breaking

Reply:
if you can tell me how to fix this, I'm happy to change, it. Right now I'm so totally burned out on this code base. If that's going to hold it up I might just quit now, also can you tell me if runtime 22.08 is acceptable

Reply:
ok I've made this change, but there's one file that I needed to copy from the debug build output to stop this from crashing.